### PR TITLE
TransformDataInternational.Qwickr version 3.4.0

### PR DESCRIPTION
**Pull request by khaffner**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115677&drop=dogfoodAlpha

### CODE REVIEW

**Comment by khaffner** [comment]
I get this issue, is this the problem on the package manager bot as well?

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-08-15 23:30:36.441 [CLI ] ShellExecute installer failed: 1603 Property(S): ErrorDialog = ErrorDlg 
MSI (s) (10:74) [23:30:35:946]: Product: Qwickr -- Installation failed. 
MSI (s) (10:74) [23:30:35:946]: Windows Installer installed the product. Product Name: Qwickr. Product Version: 3.4.0. Product Language: 1033. Manufacturer: Transform Data International. Installation success or error status: 1603. 

image

**Comment by stephengillie** [comment]
Manual Validation ended with the same error. Running the installer manually shows a `.NET Core` dependency:
image

**Comment by Trenly** [comment]
Dependencies are now supported in the 1.6 client. If you could please submit a new manifest for the latest version of the package, it would be greatly appreciated!

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 1/2/2024 6:19:16 PM
 Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   595DF4C457C8AD2DD487905020C88378BAA202DFB72307FC6404904CAEA6E4B0
 Expected hash: 16027df64a3ca2441b5d1ceb1db83fd65d2180b6f5151dc0cbf194cf80e8cb25

(Automated response - build 516.)

File: manifests/t/TransformDataInternational/Qwickr/3.4.0/TransformDataInternational.Qwickr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
+Installers:
+- InstallerUrl: https://downloads.transformdata.eu/qwickr/latest.php
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: TDI-Qwickr x64.msi
+  InstallerSha256: 16027DF64A3CA2441B5D1CEB1DB83FD65D2180B6F5151DC0CBF194CF80E8CB25
```
Comment (stephengillie) [inline manifests/t/TransformDataInternational/Qwickr/3.4.0/TransformDataInternational.Qwickr.installer.yaml:17]:
```suggestion
  InstallerSha256: 595DF4C457C8AD2DD487905020C88378BAA202DFB72307FC6404904CAEA6E4B0
```

**Pull request closed without merging**